lib/partial/audio-segment-stream.js

```diff
@@ -4,6 +4,7 @@ var Stream = require('../utils/stream.js');
 var mp4 = require('../mp4/mp4-generator.js');
 var audioFrameUtils = require('../mp4/audio-frame-utils');
 var trackInfo = require('../mp4/track-decode-info.js');
+var ONE_SECOND_IN_TS = require('../utils/clock').ONE_SECOND_IN_TS;
```

Comment:
This isn't a required change but we use it this way in most other files. I think I missed this during the original rebase.